yarn-project/aztec.js/src/contract/contract_function_interaction.ts

```diff
@@ -16,7 +24,7 @@ import type {
 /**
  * Represents the result type of a simulation.
  * By default, it will just be the return value of the simulated function
- * so contract interfaces behave as plain functions. If `includeStats` is set to true,
+ * so contract interfaces behave as plain functions. If `includeMetadata` is set to true,
```

Comment:
```suggestion
 * so contract interfaces behave as plain functions. If `includeMetadata` is set to true in `SimulateMethodOptions` on the input of `simulate(...)`,
```

Just to make it clearer where the param is set.